.github/workflows/release.yml

```diff
@@ -1,57 +1,108 @@
 name: Release
 
 on:
-  push:
-    tags:
-      - '*'
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The Infinispan branch to checkout when cutting the release."
+        required: true
+        default: "main"
+      version:
+        description: "Release version"
+        required: true
+      nextVersion:
+        description: "Next version"
+        required: true
 
 jobs:
   release:
     runs-on: ubuntu-latest
 
     outputs:
       images: ${{ steps.images.outputs.images }}
-      upload_url: ${{ steps.release.outputs.upload_url }}
 
     steps:
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '24'
+          distribution: 'temurin'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.INFINISPAN_MAVEN_GPG_ARMORED }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
       - name: Checkout Source
         uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
+          token: ${{ secrets.INFINISPAN_RELEASE_TOKEN }}
 
-      - id: images
```

Comment:
there's an output with this id defined above for this job. is still required?

Reply:
I've removed references to the `images` output